.github/workflows/build-dev.yml

```diff
@@ -0,0 +1,43 @@
+name: build-dev
+
+on: 
+  push:
+    branches:
+      - dev
+      - master
+    paths:
+      - 'src/python/requirements.txt'
```

Comment:
We can use wild card here like trigging the ci if any changes happened in files inside `src/python/*` and `src/js/*`

Reply:
I know what you mean, but in theory that's what the integration.yml workflow is for. The idea with this build-dev.yml is to only be triggered by changes that affect how the Docker image is built, that's why I've only included those four files for now, but let me know what you think :pray: